### ISSUE TITLE
Provide a binary

### ISSUE THREAD

**Issue by cool-RR** [anthonyheddings/OfficeKeyFix#5]

**Comment by daxliniere** [comment]
Thanks Ryder @stringlapse, that works a treat, Looking forward to testing on another couple machines.
All the best!
Dax Liniere

**Comment by daxliniere** [comment]
@stringlapse, does this need to be run at every startup or just once? Thank you!

**Comment by stringlapse** [comment]
@daxliniere Yes, it needs to be run every time the computer is restarted. If explorer crashes/gets killed and restarts you'll need to run it again then too. If you have a lot of startup apps explorer may not restart before the officekeyfix program exits, meaning it will be able to successfully bind the keyboard shortcuts. If this is the case you'll need to increase the sleep time and recompile it.

**Comment by daxliniere** [comment]
Ahh bummer. I put it in my Startup folder, but it obviously wipes out all the explorer windows I had left open on shutdown.
Hmm, I think I'll give this one a try to see if I have any luck: https://gist.github.com/jarpy/08b2364f577dfcfddb0fd645af6f67bd

Thanks again, Ryder!
-Dax.

**Comment by Shaftoe62** [comment]
> Ahh bummer. I put it in my Startup folder, but it obviously wipes out all the explorer windows I had left open on shutdown. Hmm, I think I'll give this one a try to see if I have any luck: https://gist.github.com/jarpy/08b2364f577dfcfddb0fd645af6f67bd
> 
> Thanks again, Ryder! -Dax.

Sadly that didn't work for me. The compiled exe however, did the job for me. thanks @stringlapse !!!!
